docs/installation/docker-compose.md

```diff
@@ -300,3 +300,9 @@ If you obtain the following error when attempting to perform `make update`:
 This is due to the fact that `db/schema.rb` is updated whenever migrations are performed. `db/schema.rb` documents the database schema, which depends on the database that you are using, its version, and when the migrations were run. It is likely that your `db/schema.rb` will diverge from the one generated by the devs.
 
 You can resolve this by changing the owner of the files to be your current user, and then running `make set-perms` afterwards when you start the containers again.
+
+### autograder logs not appearing
```

Comment:
The title in the previous section is a bit misleading because it was an error message and so it was not capitalized, but generally we want it to be capitalized, so could you update it?